app/exchanges/README.md

```diff
@@ -0,0 +1,75 @@
+# Sample Sheet Generator
+
+The sample sheets are defined through configuration in
+[/config/pipelines/](/config/pipelines/)`{pipeline}.yml` for ONT and PacBio.
+
+This configuration is parsed using
+[/app/exchanges/run_csv/](/app/exchanges/run_csv/)`{pipeline}_sample_sheet.rb` as appropriate.
+
+The required format of the sample sheets can be found on page 27 of the [SMRT Link User Guide
+v12.0](https://www.pacb.com/wp-content/uploads/SMRT_Link_User_Guide_v12.0.pdf).
+
+Once generated, the sample sheets are uploaded to SMRT Link, a test instance can be found at
+[trac-smrtlink-uat.psd.sanger.ac.uk](https://trac-smrtlink-uat.psd.sanger.ac.uk:8243/sl/runs).
```

Comment:
I tend to not include internal addresses in the repo's if I can avoid it just as a precautionary measure.